.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 0.8
+before_script:
+  - npm install -g grunt-cli
```

Comment:
I think this may have come straight from reveal.js because it is in the initial commit of the template